src/components/component-playground.jsx

```diff
@@ -525,5 +548,40 @@ module.exports = React.createClass({
       orientation: contentNode.offsetHeight > contentNode.offsetWidth ?
                    'portrait' : 'landscape'
     });
+  },
+
+  _getFilteredFixtures() {
+    var components = this.props.components;
+
+    // Get components which match search text by one or more fixture names.
+    var matchedByFixtureName = _.pick(
+        _.mapValues(components, function(fixturesObject) {
```

Comment:
Also I'll take this from @andrei-picus-hs "Don't use [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) 🚔 !" 💃